### ISSUE TITLE
Wrong error message when using `max_empty_lines` with rule `vertical_whitespace`

### ISSUE THREAD

**Issue by hossamghareeb** [realm/SwiftLint#1763]
Hello, 
My swiftlint file is configured like:
```
vertical_whitespace:
  max_empty_lines: 2
```
and it works fine but the warning message is wrong. So instead of saying `warning: Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 4. (vertical_whitespace)` it should say `warning: Vertical Whitespace Violation: Limit vertical whitespace to 2 empty lines. Currently 4. (vertical_whitespace)`

**Comment by marcelofabri** [comment]
Thanks for filling this issue, @hossamghareeb! Would you like to send a PR fixing this? 💯

**Comment by hossamghareeb** [comment]
Yeah sure, I've forked the project and will work on it. I think the auto correction as well would need fixing, as it corrects to one single line instead of the `max_empty_lines`

**Comment by marcelofabri** [comment]
Closed in #1774.
